astrix-context/src/main/java/com/avanza/astrix/context/AstrixConfigurer.java

```diff
@@ -421,7 +450,7 @@ public void set(LongBeanSetting beanSetting, AstrixBeanKey<?> beanKey, long valu
 		set(beanSetting.nameFor(beanKey), value);
 	}
 
-	void registerModule(Module module) {
+	void registerModule(com.avanza.astrix.modules.Module module) {
```

Comment:
Must the packageName be explicit here?

Reply:
Interestingly, if the import is
```java
import com.avanza.astrix.modules.*;
```
then the following compilation error happens:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0:compile (default-compile) on project astrix-context: Compilation failure: Compilation failure:
[ERROR] /Users/oskwik/projects/astrix/astrix-context/src/main/java/com/avanza/astrix/context/AstrixConfigurer.java:[100,28] reference to Module is ambiguous
[ERROR]   both interface com.avanza.astrix.modules.Module in com.avanza.astrix.modules and class java.lang.Module in java.lang match
```

But if the import is explicit, like
```java
import com.avanza.astrix.modules.Module;
```
Then the compilation succeeds.

I'll update this line to remove the package name. Thanks for noticing!

Reply:
Also, it should be noted that the `gs14` branch uses the full name including package on these lines, hence the changes (which are now reverted). One could speculate why this was done in the first place, maybe because of compilation problems as listed above? :)

Reply:
@oskarwiksten probably! Lets check if we can apply the new change back to the gs14 branch as well.